internal/pkg/deploy/cloudformation/env.go

```diff
@@ -28,35 +28,72 @@ const (
 
 // DeployAndRenderEnvironment creates the CloudFormation stack for an environment, and render the stack creation to out.
 func (cf CloudFormation) DeployAndRenderEnvironment(out progress.FileWriter, env *deploy.CreateEnvironmentInput) error {
```

Comment:
nit: should we rename this method to 
```suggestion
func (cf CloudFormation) CreateAndRenderEnvironment(out progress.FileWriter, env *deploy.CreateEnvironmentInput) error {
```